### PR TITLE
projects/rhcos: Import

### PR DESCRIPTION
**Pull request by cgwalters**
Right now we just have:

 - The `rhcos/` project where we push our oscontainers
 - The coreos-assembler build config

### CODE REVIEW

File: projects/rhcos/roles/roles.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  roleRef:
+    name: edit
+  userNames:
+    - ashcrow
+    - dm0-
+    - sdemos
+    - cgwalters
+    - yuqi-zhang
+    - miabbott
+
```
Comment (miabbott) [inline projects/rhcos/roles/roles.yaml]: Want to add @kikisdeliveryservice while we are at it?

Comment (cgwalters) [reply projects/rhcos/roles/roles.yaml]: Done, thanks!

Comment (kikisdeliveryservice) [reply projects/rhcos/roles/roles.yaml]: thanks @miabbott & @cgwalters !

**Comment by smarterclayton** [comment]
Don't forget to add these to the makefile

**Comment by cgwalters** [comment]
OK, I took a crack at updating `Makefile`.

**Comment by miabbott** [comment]
> Do our new targets need to be added to `all` or any other roll-up target?

It looks like `rhcos` could be added to the `project` target?

https://github.com/openshift/release/blob/master/Makefile#L198

And `rhcos-roles` to the `roles` target?

https://github.com/openshift/release/blob/master/Makefile#L16

**Comment by ashcrow** [comment]
Looks right to me @miabbott

File: Makefile [hunk 2]
```diff
@@ -222,6 +222,17 @@ gcsweb:
 	$(MAKE) applyTemplate WHAT=projects/gcsweb/pipeline.yaml
 .PHONY: gcsweb
 
+rhcos:
```
Comment (smarterclayton) [inline Makefile]: needs to be referenced by the `all target`

Comment (smarterclayton) [reply Makefile]: Projects is probably best.

**Comment by ashcrow** [comment]
This is out of date. Should this be closed or is this still needed in an updated and rebased form?

**Comment by ashcrow** [comment]
Closing. If we want to come back to this we can reopen or start a new PR.

**Pull request closed without merging**